### ISSUE TITLE
How to change links text near the edges?

### ISSUE THREAD

**Issue by Filyus** [giuspen/cherrytree#539]
If you type text near the edge, then it will not be a link (i.e. try to add text at the beginning or end of the link). In addition, there is no option to change the link text.

**Comment by giuspen** [comment]
You can right click, edit link

**Comment by Filyus** [comment]
I can change only URL of the link there, not the text.

**Comment by giuspen** [comment]
You're quite right. You should treat the link as any rich text property (which in fact it is) and currently the only way is to move the cursor after the first marked up char and before the last marked up char, change the text and then if necessary remove the first or last char. I may improve this in the future gtk3 version but cannot do it in the current pygtk2 version
